chat.js

```diff
@@ -1322,7 +1323,7 @@ Chat.splitFirst = function (str, delimiter, limit = 1) {
  * @param  {object} options
```

Comment:
Also we use `AnyObject` not `object` for things like this. TypeScript apparently doesn't support `object` in JSDoc.

Reply:
Or, at least, `object` gets converted to `any` for me.